pages/about.html

```diff
@@ -1,12 +1,73 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
-</head>
-<body>
-  
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Just_Jodi/about</title>
+    <link href="../stylesheets/style.css" rel="stylesheet" />
+    <link href="../stylesheets/about.css" rel="stylesheet" />
+  </head>
+  <body>
+    <header>
+      <h1 class="name">JODI DENNEY</h1>
+      <h3 class="job_title">Future Full Stacker</h3>
+      <h5 class="debut">Launching Spring 2023</h5>
+      <nav>
+        <a href="index.html">HOME</a>
+        <a href="portfolio.html"> PORTFOILO</a>
+        <a href="about.html" class="on_page"> ABOUT</a>
+      </nav>
+    </header>
+    <main>
+      <article>
+        A west coast transplant from a midwest family, I came to Seattle to go
+        to college at UW and have been here ever since. My time on this planet
+        has always revolved around stepping up for my community and stepping
+        into a challenge. I have supervised programs for homeless youth and
+        young adults, helpd trained Haiti's first EMTs, led youth activites
+        outside London and volunteered with multiple local LGBT community
+        groups. Always wanting to tackle what is hard, I dedicated myself to
+        becoming a professional firefighter. After a 16 year career and
+        thousands of 911 responses, I am pivoting from asking the most of my
+        body to asking the most of my mind. I am honored to be a member of Ada
+        Developers Adcademy's Cohort 17. My dream is to use the wealth and
+        innovation generated by today's tech industry to improve the lives of
+        our under represented neighbors. When not daydreaming about improving
+        the planet, I have found time to road trip, camp, raise the best dogs
+        and nurture the most fulfilling friendships.
+      </article>
+      <img
+        id="fire"
+        src="../images/jodi_fire.JPG"
+        alt="Jodi sitting on a fire truck"
+      />
+      <img
+        id="pride"
+        src="../images/jodi_pride.PNG"
+        alt="Jodi's face in front of a rainbow pride flag"
+      />
+      <img id="pups" src="../images/pups.png" alt="Two dogs on th beach" />
+      <section>
+        <p>LANGUAGES</p>
```

Comment:
Style: Here's where the heading should have gone:

```suggestion
        <h2>LANGUAGES</h2>
```